doc/source/reference/c-api.generalized-ufuncs.rst

```diff
@@ -1,3 +1,5 @@
+.. _c-api.generalized-ufuncs:
```

Comment:
I'd like to add a reference to this doc page from the docstring for `guvectorize`. Do we have a standard way to do that? I guess a URL under "References" is probably most robust, though I was thinking a sphinx reference might work...